corehq/apps/reports/commtrack/util.py

```diff
@@ -43,12 +43,8 @@ def sql_product(self):
     @property
     def sql_location(self):
         if self.location_id and not self._sql_location:
-            try:
-                self._sql_location = SQLLocation.objects.get(domain=self.domain, location_id=self.location_id)
-            except ObjectDoesNotExist:
-                # todo: cache this result so multiple failing calls don't keep hitting the DB
-                return None
-
+            self._sql_location = SQLLocation.objects.get_or_None(domain=self.domain, location_id=self.location_id)
```

Comment:
There is a subtle change in behaviour with this one. In the original `self._sql_location` would not be (re)set to `None` if the location didn't exist. But I think the change can't affect anything, because from the `if` statement above we know `self._sql_location` is `None` already.